sunkit_image/flct/flct.py

```diff
@@ -0,0 +1,285 @@
+        The order in which the array elements are stored that is whether they are stored as row
+        major or column major.
+        Defaults to `row`.
+    quiet : `bool`, optional
+        If set to `True` all the error messages of FLCT C code will be suppressed.
+        Defaults to `False`.
+    biascor : `bool`, optional
+        If set to `True` bias correction will be applied while computing the velocities.
+        This bias is intrinsic to the FLCT algorithm and can underestimate the velocities
+        during calculations. For more
+        details visit `here <http://solarmuri.ssl.berkeley.edu/~fisher/public/software/FLCT/C_VERSIONS/flct_1.06/doc/bias_correction_in_flct.txt>`__.
+    thresh : `float`, optional
+        A calculation will not be done for a pixel if the average absolute value
+        between the two images is less than ``thresh``.
+        If ``thresh`` is between 0 and 1, ``thresh`` is assumed given in
+        in relative units of the maximum absolute pixel value in the average of the two images.
+        Defaults to 0.
+    absflag : `bool`, optional
+        This is set to `True` to force the ``thresh`` values between 0 and 1 to be considered in
+        absolute terms.
+        Defaults to False.
+    skip : `int`, optional
+        The number of pixels to be skipped in the ``x`` and ``y`` direction after each calculation of a
+        velocity for a pixel.
+        Defaults to `None`.
+    xoff : `int`, optional
+        The offset in "x" direction after ``skip`` is enabled.
+        Defaults to 0.
+    yoff : `int`, optional
+        The offset in "y" direction after ``skip`` is enabled.
+        Defaults to 0.
+    interp : `bool`, optional
+        If set to `True` interpolation will be performed at the skipped pixels.
+        Defaults to `False`.
+    kr : `float`, optional
+        Apply a low-pass filter to the sub-images, with a Gaussian of a characteristic wavenumber
+        that is a factor of ``kr`` times the largest possible wave numbers in "x", "y" directions.
+        ``kr`` should be positive.
+        Defaults to `None`
+    pc : `bool`, optional
+        Set to `True` if the images are Plate Carrée projected.
+        Defaults to `False`.
+    latmin : `float`, optional
+        Lower latitude limit in radians, used with ``pc``.
+        Defaults to 0.
+    latmax : `float`, optional
+        Upper latitude limit in radians, used with ``pc``.
+        Defaults to 0.2.
+
+    Returns
+    -------
+    `tuple`
+        A tuple containing the velocity `~numpy.ndarray` in the following order ``vx``, ``vy``, and ``vm``.
+        ``vx`` is the velocity at every pixel location in the ``x`` direction.
+        ``vy`` is the velocity at every pixel location in the ``y`` direction.
+        ``vm`` is the mask array which is set to 1 at pixel locations where the FLCT calculations
+        were performed, 0 where the calculations were not performed and 0.5 where the results were
```

Comment:
Which results are interpolated?  When a pixel in vm has a value 0.5, does that mean that both vx and vy are interpolated?

Reply:
Yes that is what was in the C docs as far as I  remember